libr/asm/p/asm_arm_cs.c

```diff
@@ -183,6 +270,24 @@ static int assemble(RAsm *a, RAsmOp *op, const char *buf) {
 	return opsize;
 }
 
+static bool init (void* user) {
```

Comment:
im not a big fan of using globals. Also, remove space before ( ^.

Maybe it will be good to make use of *user.. or well, pass RAsm*  argument to init/fini so we can get rid of all those globals

Reply:
I agree and attempted to correct this as suggested, however doing so impacts other plugins (e.g. snes,tms320_fini,...) and requires some change in the logic of core component (e.g. plugin_free() in asm.c).

The [previous commit](https://github.com/radareorg/radare2/pull/17509/commits/693d709ee2eb91c6388995bd061ff983b83bcefd) cleans up the entries in the HTs if something is changed, resulting in low footprint.

I have seen other plugins having globals, so maybe we can go forward with this proposal (for now) and open issue to implement this change globally. What do you think ?